digitalidentity/service.go

```diff
@@ -113,3 +114,34 @@ func CreateShareQrCode(httpClient requests.HttpClient, sessionID string, clientS
 	err = json.Unmarshal(responseBytes, qrCode)
 	return qrCode, err
 }
+
+// Get Share QR info using the supplied sessionID
+func GetShareQr(httpClient requests.HttpClient, sessionID string, clientSdkId, apiUrl string, key *rsa.PrivateKey) (share ShareSessionFetchedQrCode, err error) {
+	endpoint := fmt.Sprintf(identitySessionQrCodeRetrieval, sessionID)
+	headers := requests.AuthHeader(clientSdkId)
+	request, err := requests.SignedRequest{
+		Key:        key,
+		HTTPMethod: http.MethodGet,
+		BaseURL:    apiUrl,
+		Endpoint:   endpoint,
+		Headers:    headers,
+	}.Request()
+	if err != nil {
+		return share, err
+	}
+
+	response, err := requests.Execute(httpClient, request)
+	if err != nil {
+		return share, err
+	}
+	defer response.Body.Close()
+
+	responseBytes, err := io.ReadAll(response.Body)
+	if err != nil {
+		return share, err
+	}
+
+	err = json.Unmarshal(responseBytes, &share)
```

Comment:
`share`? This as instance of `ShareSessionFetchedQrCode` should be named accordinly

Reply:
updated